sql/02-bde_control_functions.sql.in

```diff
@@ -2552,14 +2546,6 @@ BEGIN
 
     PERFORM bde_control._bde_CopyStatisticsInformation(p_upload,p_bdetable,p_tmptable);
 
-    -- Copy ownership and access rights
```

Comment:
* @strk , the same question. Why removal of this is needed ?

Reply:
This used to be needed when we used the DROP/REPLACE approach: we wanted the temporary table to have the _same_ ownership and access rights as the original table.

Now that we directly store the data in the original table we dont' need the copy. There might be even more things we can avoid doing nowadays, but for this PR this is a required change, because ownership of BDE tables is to `bde_dba` and a user who does NOT have `bde_dba` among its privileges can NOT give ownership to that user.

This was explained in the relevant commit: https://github.com/linz/linz-bde-uploader/pull/249/commits/d8c04c3a8730d6eb9d476f440940b5604375d817

Reply:
Thanks, I understand now.